NitroxServer/GameLogic/Bases/BuildingManager.cs

```diff
@@ -102,10 +102,26 @@ public bool ModifyConstructedAmount(ModifyConstructedAmount modifyConstructedAmo
             Log.Error($"Trying to modify the constructed amount of a non-registered object (GhostId: {modifyConstructedAmount.GhostId})");
             return false;
         }
+
+        // Certain entities with a Constructable are just "regular" WorldEntities (e.g. starship boxes) and for simplicity we'll just not persist their progress
+        // since their only use is to be deconstructed to give materials to players
```

Comment:
Do we want to raise a ticket for someone to do this one day?

Reply:
That's up to you. I could definitely implement it in no time but it'd only be overkill/making the base system persistence a bit more confusing.